content-api/content-service/conf/routes

```diff
@@ -30,7 +30,7 @@ DELETE    /content/v3/hierarchy/remove      controllers.v3.ContentController.rem
 PATCH     /content/v3/hierarchy/update      controllers.v3.ContentController.updateHierarchy
 GET       /content/v3/hierarchy/:identifier      controllers.v3.ContentController.getHierarchy(identifier:String, mode:Option[String])
 POST      /content/v3/upload/:identifier         controllers.v3.ContentController.upload(identifier:String)
-POST      /content/v3/copy/:identifier      controllers.v3.ContentController.copy(identifier:String, mode:Option[String])
+POST      /content/v3/copy/:identifier      controllers.v3.ContentController.copy(identifier:String, mode:Option[String], copyType:String ?= "deep")
```

Comment:
the path parameter should be `type` not `copyType`. 
Ex: /content/v3/copy/do_123?type=shallow

Reply:
done